install.sh

```diff
@@ -65,13 +83,15 @@ export LP_DIR=$ALIGN_HOME/lpsolve
 #export BOOST_LP=$ALIGN_HOME/boost
 export JSON=$ALIGN_HOME/json
 export GTEST_DIR=$ALIGN_HOME/googletest/googletest/
-export VENV=$ALIGN_HOME/general
```

Comment:
I think we want to leave the VENV as default.

Reply:
What about using the VENV automatically from within the Makefile then? I dont like it that much that you always have to specify the VENV when calling make, or does this has a helpful purpose that I just didnt understood? I would prefer ALIGN to be usable both with and without a VENV

Reply:
We don't use the Makefile much any more. We prefer the command line form (schematic2layout.py). The reason we have a VENV= argument to make is because people had named their environments differently. The main reason  we use virtual environments (other than not contaminating your machine with possibly python libraries) is so we can use "python" and "pip" instead of "python3" and "pip3" on the command line. We can try to make everything work so that it doesn't use the virtual environment if the VENV environment variable is not set. Is that reasonable?